__tests__/app.test.js

```diff
@@ -110,17 +110,71 @@ test('400 status response with an  invalid article ID', () => {
 
 
 
-describe('GET /api/comments/articleId/count', () => {
+describe('GET /api/comments/:article_id', () => {
+ 
   test('should return a status :200 and return a body containing an array of objects for the comments with a particular article Id', () => {
+    const article_id = 3;
     return request(app)
-    .get('/api/comments/2/count')
+    .get(`/api/comments/${article_id}`)
     .expect(200)
-    .then(({body}) => {
-      console.log(body)
-    })
+    .then(({body}) => { 
+     for (let i = 0; i < body.length; i++) {     
```

Comment:
this for loop is completely unnecessary to use with a length test inside, but you might test the length first and then forEach through each comment that is contains the expected properties

Reply:
OK will remove the loop.